spec/services/displayable_pii_formatter_spec.rb

```diff
@@ -168,15 +168,15 @@
         let(:dob) { '1990-01-02' }
 
         it 'returns a formatted birdate' do
-          expect(formatter.format.birthdate).to eq('January 02, 1990')
+          expect(formatter.format.birthdate).to eq('January 2, 1990')
```

Comment:
I don't really have strong feelings one way or the other, but it occurs to me that hard-coding the string value creates some burden for keeping the specs in sync, and we have no test coverage specific to the issue that we're addressing here. We could consider comparing this to the result of the `I18n.l` call, like we do with `verified_at` at line 111 above. On the other hand, that essentially duplicates the logic itself, and maybe it's a good thing for us to be spelling out how we want it to look in readable form.

We could consider adding a context block for a non-English locale to provide some test coverage for the fix?

```suggestion
          expect(formatter.format.birthdate).to eq('January 2, 1990')
        end

        context 'with a non-English locale' do
          before { I18n.locale = :es }

          it 'returns a localized, formatted birdate' do
            expect(formatter.format.birthdate).to eq('2 de enero de 1990')
          end
```